dcs/action.py

```diff
@@ -11,7 +12,7 @@ def __repr__(self):
         s = []
         for x in self.params:
             if isinstance(x, String):
-                s.append('getValueDictByKey("' + x.id + '")')
+                s.append("getValueDictByKey({})".format(dumps(x.id)))
             else:
                 s.append(str(x))
```

Comment:
Could you elaborate - what type `parameters` would be? I didn't find out the type for it, I assume it would be an condition instance?

Reply:
you mean the self.params?
that can be pretty any type, for example for the MessageToCoalition it is
coalition identifier: the coalition (red, blue, ...)
String() to display
number of seconds how long the message is displayed
bool if the text view should be cleared

Reply:
I mean, than the str(x) should be changed to dumps too!

Reply:
you maybe right, but I think here it is not a big problem because the string that will be added here, are pretty static